File: tests/testers/test-html/cgi-bin/php/environment.php
#!/usr/bin/php-cgi

<!DOCTYPE html>
<html>
<head>
    <title>CGI Test PHP 1 - ENV</title>
</head>
<body>
    <h1>CGI Test PHP 1 - ENV</h1>

    <?php foreach ($_SERVER as $key => $value): ?>
        <p><?= str_pad(htmlspecialchars($key), 20) ?> = <?= htmlspecialchars($value) ?></p>
    <?php endforeach; ?>

</body>
</html>

File: tests/testers/test-html/cgi-bin/py/environment.py
#!/usr/bin/python3

import os

print("Content-type: text/html\r\n")
print("<html>")
print("<head>")
print("<title>CGI Test Python 1 - ENV</title>")
print("</head>")
print("<body>")
print("<h1>CGI Test Python 1 - ENV</h1>")
for param in os.environ.keys():
    print("<p>%20s = %s</p>" % (param, os.environ[param]))
print("</body>")
print("</html>")
